frontend/src/components/QueueTable.js

```diff
@@ -58,9 +65,21 @@ function QueueTable({ queueData: { name, rows } }) {
   return (
     <Paper className={classes.root}>
       <MaterialTable
-        title={name}
+        title={queueStore.queueStats[queueData["queueData"]].name}
         className={classes.table}
-        data={rows}
+        options={{
+          search: false,
+        }}
+        data={queueStore.queues[queueData["queueData"]].map(x => ({
+          urgency: 1,
```

Comment:
are they all urgency 1?

Reply:
To be honest, I'm hazy on where we are on urgency

Reply:
so its not pulled from the backend queues table?

Reply:
I fixed it.

Reply:
where?

Reply:
QueueTable.js

  data={queueStore.queues[queueData["queueData"]].map(x => ({
          urgency: x.urgency,
          last: x.participant.last_name,
          uid: x.participant.pp_id,
          timeElapsed: moment(x.status.created_at).format("LT"),
          status: x.status.event_type,
          service: x.service.name,
          seen: false,
          Notes: false,
        }))}